app/views/courses/show.html.erb

```diff
@@ -57,7 +57,18 @@
                   <p><%= @course.description %></p>
                   <p>&nbsp;</p>
                   <h4>Prereqs:</h4>
-                  <p>TODO</p>
+                  <p>
+                  <ul>
+                    <% prereq_sets = @course.prerequisites %>
+                    <% if prereq_sets.empty? %>
+                      <li>None</li>
+                    <% else %>
+                      <% @course.prerequisites.each do |prereq_set| %>
+                        <li><%= prereq_set.map(&:to_s).join(" or ") %></li>
+                      <% end%>
+                    <% end %>
```

Comment:
Seems like too much logic for the view layer, perhaps move most of this to a helper method?

i.e. something like this:
```
def prereq_sets(course)
  if prereq_sets.empty?
    concat content_tag(:li, "None")
  else
    course.prerequisites.each do |prereq_set|
      concat content_tag(:li, prereq_set.map(&:to_s).join(" or "))
    end
  end
end
```